.claude/skills/pr-flow/SKILL.md

```
@@ -1,350 +1,311 @@
+#### For each inline comment:
+1. Read the full comment body
+2. Assess: is the issue valid?
+   - **Valid**: Fix the code, reply with commit SHA and what changed
+   - **Disagree**: Reply explaining why the current approach is correct
+3. Reply via `gh api repos/{owner}/{repo}/pulls/comments/<ID>/replies -f body="..."`
+
+#### For outside-diff comments:
+1. These appear in the review body, not as inline threads
+2. **IMPORTANT**: When posting PR comments (non-direct thread replies), always include `@coderabbitai` mention so CodeRabbit can detect and track the response
+3. Address them in a PR comment summarizing all fixes:
+   ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)